### PR TITLE
backport of CLOUDSTACK-8197 create MinVRVersion config item

### PR DESCRIPTION
**Pull request by DaanHoogland**
Conflicts:
  api/src/com/cloud/network/VirtualNetworkApplianceService.java
  engine/orchestration/src/org/apache/cloudstack/engine/orchestration/NetworkOrchestrator.java

### CODE REVIEW

File: engine/api/src/org/apache/cloudstack/engine/orchestration/service/NetworkOrchestrationService.java [hunk 1]
```diff
@@ -62,9 +63,13 @@
     static final String NetworkLockTimeoutCK = "network.lock.timeout";
     static final String GuestDomainSuffixCK = "guest.domain.suffix";
     static final String NetworkThrottlingRateCK = "network.throttling.rate";
+    static final String MinVRVersionCK = "minreq.sysvmtemplate.version";
+
+    static final ConfigKey<String> MinVRVersion = new ConfigKey<String>(String.class, MinVRVersionCK, "Advanced", "4.5.2",
```
Comment (bhaisaab) [inline engine/api/src/org/apache/cloudstack/engine/orchestration/service/NetworkOrchestrationService.java]: I guess all 4.5.x releases should support 4.5.0 as minimum VR version? so can we change this to 4.5.0?

Comment (DaanHoogland) [inline engine/api/src/org/apache/cloudstack/engine/orchestration/service/NetworkOrchestrationService.java]: Fine

**Comment by DaanHoogland** [comment]
adjusted default to 4.5.0

**Comment by bhaisaab** [comment]
@DaanHoogland looks like asfgit failed to close both #243 and #244 can you please close it manually; merged both of them on 4.4 and 4.5 branches. Thanks.

**Pull request closed without merging**